profile/2017-03-09-profile.json

```diff
@@ -0,0 +1,47 @@
+{
+    "resource-manager": {
+        "microsoft.authorization": {
+            "version": "2015-07-01",
+            "types": {
+                "locks":"2015-01-01",
+                "policyassignments": "2015-10-01-preview",
+                "policydefinitions": "2015-10-01-preview"
+            }
+        },
+        "microsoft.compute": {
+            "version": "2016-03-30",
+            "types": {}
+        },
+        "microsoft.keyvault": {
+            "version": "2016-10-01",
+            "types": {}
+        },
+        "microsoft.network": {
+            "version": "2015-06-15",
+            "types": {}
+        },
+        "microsoft.storage": {
+            "version": "2016-01-01",
+            "types": {}
+        },
+        "microsoft.subscription": {
```

Comment:
This is false, `Microsoft.Subscription` exists, but it's not what you mean here:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/subscriptions.json#L19

This is tough because the URL is `https://management.azure.com/subscriptions?api-version=2016-06-01` that does not involve RP.
When we list providers, how this one is listed?

Reply:
If you do `az provider list`, you get:
```json
    "namespace": "Microsoft.Resources",
...
        "apiVersions": [
          "2018-02-01",
          "2018-01-01",
          "2017-08-01",
          "2017-06-01",
          "2017-05-10",
          "2017-05-01",
          "2017-03-01",
          "2016-09-01",
          "2016-07-01",
          "2016-06-01",
          "2016-02-01",
          "2015-11-01",
          "2015-01-01",
          "2014-04-01-preview"
        ],
        "locations": [],
        "properties": null,
        "resourceType": "subscriptions"
```
which is probably what I would use for consistency